tests/TestCase/View/Form/FormContextTest.php

```diff
@@ -259,13 +259,9 @@ public function testError()
         $this->assertEquals([], $context->error('Alias.name'));
         $this->assertEquals([], $context->error('nope.nope'));
 
-        $mock = $this->getMockBuilder('Cake\Validation\Validator')
-            ->setMethods(['errors'])
-            ->getMock();
-        $mock->expects($this->once())
-            ->method('errors')
-            ->willReturn(['key' => 'should be an array, not a string']);
-        $form->setValidator('default', $mock);
+        $validator = new Validator();
+        $validator->requirePresence('key', true, 'should be an array, not a string');
+        $form->setValidator('default', $validator);
```

Comment:
I couldn't suss out why this new validator was mocked, so I replaced it with a concrete validator so it implements the `->count` method relied on by the Form class.